redaxo/src/addons/media_manager/lang/de_de.lang

```diff
@@ -94,7 +98,7 @@ media_manager_effect_rounded_corners_bottomright = Unten Rechts
 
 media_manager_effect_flip = Richtung
 
-media_manager_error_type_name_not_unique = Ein Mediatyp mit diesem Namen existiert bereits!
+media_manager_error_type_name_not_unique = Ein Mediatyp mit diesem namen existiert bereits!
```

Comment:
passt so wie es ist. einfach in dem branch der für diesen PR angelegt wurde, also `phoebusryan:mediamanager-patch1` committen. jeder weitere commit wird dem PR hinzugefügt.

wir können vorm/beim mergen dann die historie wieder aufräumen.